PhiferTristan/numba-1-declarative-client-side
# Added functionality to edit button on /haulingships page

The edit button in the hauler listings on /haulingships view is now clickable and redirects to an edit form page.

## Steps to Test

1. Pull the 'TP-edit-button' branch and switch to that branch.
2. Visit the /haulingships route
3. Verify all the haulers are listed with correct info and click the 'edit' button
4. Verify the hauler you are viewing has the correct info upon initial render
5. Make changes and click the update button
6. You should be re-directed to the /haulingships view where you can see your update made to the hauler
